<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />

<meta name="author" content="L Collado-Torres" />

<meta name="date" content="2016-04-11" />

<title>Example report using bumphunter results</title>


<style type="text/css">code{white-space: pre;}</style>
<link href="data:text/css;charset=utf-8,pre%20%2Eoperator%2C%0Apre%20%2Eparen%20%7B%0Acolor%3A%20rgb%28104%2C%20118%2C%20135%29%0A%7D%0Apre%20%2Eliteral%20%7B%0Acolor%3A%20%23990073%0A%7D%0Apre%20%2Enumber%20%7B%0Acolor%3A%20%23099%3B%0A%7D%0Apre%20%2Ecomment%20%7B%0Acolor%3A%20%23998%3B%0Afont%2Dstyle%3A%20italic%0A%7D%0Apre%20%2Ekeyword%20%7B%0Acolor%3A%20%23900%3B%0Afont%2Dweight%3A%20bold%0A%7D%0Apre%20%2Eidentifier%20%7B%0Acolor%3A%20rgb%280%2C%200%2C%200%29%3B%0A%7D%0Apre%20%2Estring%20%7B%0Acolor%3A%20%23d14%3B%0A%7D%0A" rel="stylesheet" type="text/css" />
<script src="data:application/x-javascript;base64,
var hljs=new function(){function m(p){return p.replace(/&/gm,"&amp;").replace(/</gm,"&lt;")}function f(r,q,p){return RegExp(q,"m"+(r.cI?"i":"")+(p?"g":""))}function b(r){for(var p=0;p<r.childNodes.length;p++){var q=r.childNodes[p];if(q.nodeName=="CODE"){return q}if(!(q.nodeType==3&&q.nodeValue.match(/\s+/))){break}}}function h(t,s){var p="";for(var r=0;r<t.childNodes.length;r++){if(t.childNodes[r].nodeType==3){var q=t.childNodes[r].nodeValue;if(s){q=q.replace(/\n/g,"")}p+=q}else{if(t.childNodes[r].nodeName=="BR"){p+="\n"}else{p+=h(t.childNodes[r])}}}if(/MSIE [678]/.test(navigator.userAgent)){p=p.replace(/\r/g,"\n")}return p}function a(s){var r=s.className.split(/\s+/);r=r.concat(s.parentNode.className.split(/\s+/));for(var q=0;q<r.length;q++){var p=r[q].replace(/^language-/,"");if(e[p]){return p}}}function c(q){var p=[];(function(s,t){for(var r=0;r<s.childNodes.length;r++){if(s.childNodes[r].nodeType==3){t+=s.childNodes[r].nodeValue.length}else{if(s.childNodes[r].nodeName=="BR"){t+=1}else{if(s.childNodes[r].nodeType==1){p.push({event:"start",offset:t,node:s.childNodes[r]});t=arguments.callee(s.childNodes[r],t);p.push({event:"stop",offset:t,node:s.childNodes[r]})}}}}return t})(q,0);return p}function k(y,w,x){var q=0;var z="";var s=[];function u(){if(y.length&&w.length){if(y[0].offset!=w[0].offset){return(y[0].offset<w[0].offset)?y:w}else{return w[0].event=="start"?y:w}}else{return y.length?y:w}}function t(D){var A="<"+D.nodeName.toLowerCase();for(var B=0;B<D.attributes.length;B++){var C=D.attributes[B];A+=" "+C.nodeName.toLowerCase();if(C.value!==undefined&&C.value!==false&&C.value!==null){A+='="'+m(C.value)+'"'}}return A+">"}while(y.length||w.length){var v=u().splice(0,1)[0];z+=m(x.substr(q,v.offset-q));q=v.offset;if(v.event=="start"){z+=t(v.node);s.push(v.node)}else{if(v.event=="stop"){var p,r=s.length;do{r--;p=s[r];z+=("</"+p.nodeName.toLowerCase()+">")}while(p!=v.node);s.splice(r,1);while(r<s.length){z+=t(s[r]);r++}}}}return z+m(x.substr(q))}function j(){function q(x,y,v){if(x.compiled){return}var u;var s=[];if(x.k){x.lR=f(y,x.l||hljs.IR,true);for(var w in x.k){if(!x.k.hasOwnProperty(w)){continue}if(x.k[w] instanceof Object){u=x.k[w]}else{u=x.k;w="keyword"}for(var r in u){if(!u.hasOwnProperty(r)){continue}x.k[r]=[w,u[r]];s.push(r)}}}if(!v){if(x.bWK){x.b="\\b("+s.join("|")+")\\s"}x.bR=f(y,x.b?x.b:"\\B|\\b");if(!x.e&&!x.eW){x.e="\\B|\\b"}if(x.e){x.eR=f(y,x.e)}}if(x.i){x.iR=f(y,x.i)}if(x.r===undefined){x.r=1}if(!x.c){x.c=[]}x.compiled=true;for(var t=0;t<x.c.length;t++){if(x.c[t]=="self"){x.c[t]=x}q(x.c[t],y,false)}if(x.starts){q(x.starts,y,false)}}for(var p in e){if(!e.hasOwnProperty(p)){continue}q(e[p].dM,e[p],true)}}function d(B,C){if(!j.called){j();j.called=true}function q(r,M){for(var L=0;L<M.c.length;L++){if((M.c[L].bR.exec(r)||[null])[0]==r){return M.c[L]}}}function v(L,r){if(D[L].e&&D[L].eR.test(r)){return 1}if(D[L].eW){var M=v(L-1,r);return M?M+1:0}return 0}function w(r,L){return L.i&&L.iR.test(r)}function K(N,O){var M=[];for(var L=0;L<N.c.length;L++){M.push(N.c[L].b)}var r=D.length-1;do{if(D[r].e){M.push(D[r].e)}r--}while(D[r+1].eW);if(N.i){M.push(N.i)}return f(O,M.join("|"),true)}function p(M,L){var N=D[D.length-1];if(!N.t){N.t=K(N,E)}N.t.lastIndex=L;var r=N.t.exec(M);return r?[M.substr(L,r.index-L),r[0],false]:[M.substr(L),"",true]}function z(N,r){var L=E.cI?r[0].toLowerCase():r[0];var M=N.k[L];if(M&&M instanceof Array){return M}return false}function F(L,P){L=m(L);if(!P.k){return L}var r="";var O=0;P.lR.lastIndex=0;var M=P.lR.exec(L);while(M){r+=L.substr(O,M.index-O);var N=z(P,M);if(N){x+=N[1];r+='<span class="'+N[0]+'">'+M[0]+"</span>"}else{r+=M[0]}O=P.lR.lastIndex;M=P.lR.exec(L)}return r+L.substr(O,L.length-O)}function J(L,M){if(M.sL&&e[M.sL]){var r=d(M.sL,L);x+=r.keyword_count;return r.value}else{return F(L,M)}}function I(M,r){var L=M.cN?'<span class="'+M.cN+'">':"";if(M.rB){y+=L;M.buffer=""}else{if(M.eB){y+=m(r)+L;M.buffer=""}else{y+=L;M.buffer=r}}D.push(M);A+=M.r}function G(N,M,Q){var R=D[D.length-1];if(Q){y+=J(R.buffer+N,R);return false}var P=q(M,R);if(P){y+=J(R.buffer+N,R);I(P,M);return P.rB}var L=v(D.length-1,M);if(L){var O=R.cN?"</span>":"";if(R.rE){y+=J(R.buffer+N,R)+O}else{if(R.eE){y+=J(R.buffer+N,R)+O+m(M)}else{y+=J(R.buffer+N+M,R)+O}}while(L>1){O=D[D.length-2].cN?"</span>":"";y+=O;L--;D.length--}var r=D[D.length-1];D.length--;D[D.length-1].buffer="";if(r.starts){I(r.starts,"")}return R.rE}if(w(M,R)){throw"Illegal"}}var E=e[B];var D=[E.dM];var A=0;var x=0;var y="";try{var s,u=0;E.dM.buffer="";do{s=p(C,u);var t=G(s[0],s[1],s[2]);u+=s[0].length;if(!t){u+=s[1].length}}while(!s[2]);if(D.length>1){throw"Illegal"}return{r:A,keyword_count:x,value:y}}catch(H){if(H=="Illegal"){return{r:0,keyword_count:0,value:m(C)}}else{throw H}}}function g(t){var p={keyword_count:0,r:0,value:m(t)};var r=p;for(var q in e){if(!e.hasOwnProperty(q)){continue}var s=d(q,t);s.language=q;if(s.keyword_count+s.r>r.keyword_count+r.r){r=s}if(s.keyword_count+s.r>p.keyword_count+p.r){r=p;p=s}}if(r.language){p.second_best=r}return p}function i(r,q,p){if(q){r=r.replace(/^((<[^>]+>|\t)+)/gm,function(t,w,v,u){return w.replace(/\t/g,q)})}if(p){r=r.replace(/\n/g,"<br>")}return r}function n(t,w,r){var x=h(t,r);var v=a(t);var y,s;if(v){y=d(v,x)}else{return}var q=c(t);if(q.length){s=document.createElement("pre");s.innerHTML=y.value;y.value=k(q,c(s),x)}y.value=i(y.value,w,r);var u=t.className;if(!u.match("(\\s|^)(language-)?"+v+"(\\s|$)")){u=u?(u+" "+v):v}if(/MSIE [678]/.test(navigator.userAgent)&&t.tagName=="CODE"&&t.parentNode.tagName=="PRE"){s=t.parentNode;var p=document.createElement("div");p.innerHTML="<pre><code>"+y.value+"</code></pre>";t=p.firstChild.firstChild;p.firstChild.cN=s.cN;s.parentNode.replaceChild(p.firstChild,s)}else{t.innerHTML=y.value}t.className=u;t.result={language:v,kw:y.keyword_count,re:y.r};if(y.second_best){t.second_best={language:y.second_best.language,kw:y.second_best.keyword_count,re:y.second_best.r}}}function o(){if(o.called){return}o.called=true;var r=document.getElementsByTagName("pre");for(var p=0;p<r.length;p++){var q=b(r[p]);if(q){n(q,hljs.tabReplace)}}}function l(){if(window.addEventListener){window.addEventListener("DOMContentLoaded",o,false);window.addEventListener("load",o,false)}else{if(window.attachEvent){window.attachEvent("onload",o)}else{window.onload=o}}}var e={};this.LANGUAGES=e;this.highlight=d;this.highlightAuto=g;this.fixMarkup=i;this.highlightBlock=n;this.initHighlighting=o;this.initHighlightingOnLoad=l;this.IR="[a-zA-Z][a-zA-Z0-9_]*";this.UIR="[a-zA-Z_][a-zA-Z0-9_]*";this.NR="\\b\\d+(\\.\\d+)?";this.CNR="\\b(0[xX][a-fA-F0-9]+|(\\d+(\\.\\d*)?|\\.\\d+)([eE][-+]?\\d+)?)";this.BNR="\\b(0b[01]+)";this.RSR="!|!=|!==|%|%=|&|&&|&=|\\*|\\*=|\\+|\\+=|,|\\.|-|-=|/|/=|:|;|<|<<|<<=|<=|=|==|===|>|>=|>>|>>=|>>>|>>>=|\\?|\\[|\\{|\\(|\\^|\\^=|\\||\\|=|\\|\\||~";this.ER="(?![\\s\\S])";this.BE={b:"\\\\.",r:0};this.ASM={cN:"string",b:"'",e:"'",i:"\\n",c:[this.BE],r:0};this.QSM={cN:"string",b:'"',e:'"',i:"\\n",c:[this.BE],r:0};this.CLCM={cN:"comment",b:"//",e:"$"};this.CBLCLM={cN:"comment",b:"/\\*",e:"\\*/"};this.HCM={cN:"comment",b:"#",e:"$"};this.NM={cN:"number",b:this.NR,r:0};this.CNM={cN:"number",b:this.CNR,r:0};this.BNM={cN:"number",b:this.BNR,r:0};this.inherit=function(r,s){var p={};for(var q in r){p[q]=r[q]}if(s){for(var q in s){p[q]=s[q]}}return p}}();hljs.LANGUAGES.cpp=function(){var a={keyword:{"false":1,"int":1,"float":1,"while":1,"private":1,"char":1,"catch":1,"export":1,virtual:1,operator:2,sizeof:2,dynamic_cast:2,typedef:2,const_cast:2,"const":1,struct:1,"for":1,static_cast:2,union:1,namespace:1,unsigned:1,"long":1,"throw":1,"volatile":2,"static":1,"protected":1,bool:1,template:1,mutable:1,"if":1,"public":1,friend:2,"do":1,"return":1,"goto":1,auto:1,"void":2,"enum":1,"else":1,"break":1,"new":1,extern:1,using:1,"true":1,"class":1,asm:1,"case":1,typeid:1,"short":1,reinterpret_cast:2,"default":1,"double":1,register:1,explicit:1,signed:1,typename:1,"try":1,"this":1,"switch":1,"continue":1,wchar_t:1,inline:1,"delete":1,alignof:1,char16_t:1,char32_t:1,constexpr:1,decltype:1,noexcept:1,nullptr:1,static_assert:1,thread_local:1,restrict:1,_Bool:1,complex:1},built_in:{std:1,string:1,cin:1,cout:1,cerr:1,clog:1,stringstream:1,istringstream:1,ostringstream:1,auto_ptr:1,deque:1,list:1,queue:1,stack:1,vector:1,map:1,set:1,bitset:1,multiset:1,multimap:1,unordered_set:1,unordered_map:1,unordered_multiset:1,unordered_multimap:1,array:1,shared_ptr:1}};return{dM:{k:a,i:"</",c:[hljs.CLCM,hljs.CBLCLM,hljs.QSM,{cN:"string",b:"'\\\\?.",e:"'",i:"."},{cN:"number",b:"\\b(\\d+(\\.\\d*)?|\\.\\d+)(u|U|l|L|ul|UL|f|F)"},hljs.CNM,{cN:"preprocessor",b:"#",e:"$"},{cN:"stl_container",b:"\\b(deque|list|queue|stack|vector|map|set|bitset|multiset|multimap|unordered_map|unordered_set|unordered_multiset|unordered_multimap|array)\\s*<",e:">",k:a,r:10,c:["self"]}]}}}();hljs.LANGUAGES.r={dM:{c:[hljs.HCM,{cN:"number",b:"\\b0[xX][0-9a-fA-F]+[Li]?\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\b\\d+(?:[eE][+\\-]?\\d*)?L\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\b\\d+\\.(?!\\d)(?:i\\b)?",e:hljs.IMMEDIATE_RE,r:1},{cN:"number",b:"\\b\\d+(?:\\.\\d*)?(?:[eE][+\\-]?\\d*)?i?\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"number",b:"\\.\\d+(?:[eE][+\\-]?\\d*)?i?\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"keyword",b:"(?:tryCatch|library|setGeneric|setGroupGeneric)\\b",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\.\\.\\.",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\.\\.\\d+(?![\\w.])",e:hljs.IMMEDIATE_RE,r:10},{cN:"keyword",b:"\\b(?:function)",e:hljs.IMMEDIATE_RE,r:2},{cN:"keyword",b:"(?:if|in|break|next|repeat|else|for|return|switch|while|try|stop|warning|require|attach|detach|source|setMethod|setClass)\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"literal",b:"(?:NA|NA_integer_|NA_real_|NA_character_|NA_complex_)\\b",e:hljs.IMMEDIATE_RE,r:10},{cN:"literal",b:"(?:NULL|TRUE|FALSE|T|F|Inf|NaN)\\b",e:hljs.IMMEDIATE_RE,r:1},{cN:"identifier",b:"[a-zA-Z.][a-zA-Z0-9._]*\\b",e:hljs.IMMEDIATE_RE,r:0},{cN:"operator",b:"<\\-(?!\\s*\\d)",e:hljs.IMMEDIATE_RE,r:2},{cN:"operator",b:"\\->|<\\-",e:hljs.IMMEDIATE_RE,r:1},{cN:"operator",b:"%%|~",e:hljs.IMMEDIATE_RE},{cN:"operator",b:">=|<=|==|!=|\\|\\||&&|=|\\+|\\-|\\*|/|\\^|>|<|!|&|\\||\\$|:",e:hljs.IMMEDIATE_RE,r:0},{cN:"operator",b:"%",e:"%",i:"\\n",r:1},{cN:"identifier",b:"`",e:"`",r:0},{cN:"string",b:'"',e:'"',c:[hljs.BE],r:0},{cN:"string",b:"'",e:"'",c:[hljs.BE],r:0},{cN:"paren",b:"[[({\\])}]",e:hljs.IMMEDIATE_RE,r:0}]}};
hljs.initHighlightingOnLoad();

"></script>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>


<link href="data:text/css;charset=utf-8,body%2C%20td%20%7B%0Afont%2Dfamily%3A%20sans%2Dserif%3B%0Abackground%2Dcolor%3A%20white%3B%0Afont%2Dsize%3A%2013px%3B%0A%7D%0Abody%20%7B%0Amax%2Dwidth%3A%20800px%3B%0Amargin%3A%200%20auto%3B%0Apadding%3A%201em%201em%202em%3B%0Aline%2Dheight%3A%2020px%3B%0A%7D%0A%0Adiv%23TOC%20li%20%7B%0Alist%2Dstyle%3Anone%3B%0Abackground%2Dimage%3Anone%3B%0Abackground%2Drepeat%3Anone%3B%0Abackground%2Dposition%3A0%3B%0A%7D%0A%0Ap%2C%20pre%20%7B%20margin%3A%200em%200em%201em%3B%0A%7D%0A%0Aimg%2C%20table%20%7B%0Amargin%3A%200em%20auto%201em%3B%0A%7D%0Ap%20%7B%0Atext%2Dalign%3A%20justify%3B%0A%7D%0Att%2C%20code%2C%20pre%20%7B%0Afont%2Dfamily%3A%20%27DejaVu%20Sans%20Mono%27%2C%20%27Droid%20Sans%20Mono%27%2C%20%27Lucida%20Console%27%2C%20Consolas%2C%20Monaco%2C%20monospace%3B%0A%7D%0Ah1%2C%20h2%2C%20h3%2C%20h4%2C%20h5%2C%20h6%20%7B%20font%2Dfamily%3A%20Helvetica%2C%20Arial%2C%20sans%2Dserif%3B%0Amargin%3A%201%2E2em%200em%200%2E6em%200em%3B%0Afont%2Dweight%3A%20bold%3B%0A%7D%0Ah1%2Etitle%20%7B%0Afont%2Dsize%3A%20250%25%3B%0Afont%2Dweight%3A%20normal%3B%0Acolor%3A%20%2387b13f%3B%0Aline%2Dheight%3A%201%2E1em%3B%0Amargin%2Dtop%3A%200px%3B%0Aborder%2Dbottom%3A%200px%3B%0A%7D%0Ah1%20%7B%0Afont%2Dsize%3A%20160%25%3B%0Afont%2Dweight%3A%20normal%3B%0Aline%2Dheight%3A%201%2E4em%3B%0Aborder%2Dbottom%3A%201px%20%231a81c2%20solid%3B%0A%7D%0Ah2%20%7B%0Afont%2Dsize%3A%20130%25%3B%20%7D%0Ah1%2C%20h2%2C%20h3%20%7B%0Acolor%3A%20%231a81c2%3B%0A%7D%0Ah3%2C%20h4%2C%20h5%2C%20h6%20%7B%0Afont%2Dsize%3A115%25%3B%0A%7D%20%0A%0Aa%20%7B%20color%3A%20%231a81c2%3B%20%7D%0Aa%3Aactive%20%7B%20outline%3A%20none%3B%20%7D%0Aa%3Avisited%20%7B%20color%3A%20%231a81c2%3B%20%7D%0Aa%3Ahover%20%7B%20color%3A%20%234c94c2%3B%20%7D%0Apre%2C%20img%20%7B%0Amax%2Dwidth%3A%20100%25%3B%0Adisplay%3A%20block%3B%0A%7D%0Apre%20%7B%0Aborder%3A%200px%20none%3B%0Abackground%2Dcolor%3A%20%23F8F8F8%3B%0Awhite%2Dspace%3A%20pre%3B%0Aoverflow%2Dx%3A%20auto%3B%0A%7D%0Apre%20code%20%7B%0Aborder%3A%201px%20%23aaa%20dashed%3B%0Abackground%2Dcolor%3A%20white%3B%0Adisplay%3A%20block%3B%0Apadding%3A%201em%3B%20color%3A%20%23111%3B%0Aoverflow%2Dx%3A%20inherit%3B%0A%7D%0A%0Apre%20code%5Bclass%5D%20%7B%0Abackground%2Dcolor%3A%20inherit%3B%0A%7D%0A%0Apre%5Bclass%5D%20code%20%7B%0Abackground%2Dcolor%3A%20inherit%3B%0A%7D%0A%0Acode%20%7B%20background%2Dcolor%3A%20transparent%3B%0Acolor%3A%20%2387b13f%3B%0Afont%2Dsize%3A%2092%25%3B%0A%7D%0A%0Atable%2C%20td%2C%20th%20%7B%0Aborder%3A%20none%3B%0Apadding%3A%200%200%2E5em%3B%0A%7D%0A%0Atbody%20tr%3Anth%2Dchild%28odd%29%20td%20%7B%0Abackground%2Dcolor%3A%20%23F8F8F8%3B%0A%7D%0Ablockquote%20%7B%0Acolor%3A%23666666%3B%0Amargin%3A0%3B%0Apadding%2Dleft%3A%201em%3B%0Aborder%2Dleft%3A%200%2E5em%20%23EEE%20solid%3B%0A%7D%0Ahr%20%7B%0Aheight%3A%200px%3B%0Aborder%2Dbottom%3A%20none%3B%0Aborder%2Dtop%2Dwidth%3A%20thin%3B%0Aborder%2Dtop%2Dstyle%3A%20dotted%3B%0Aborder%2Dtop%2Dcolor%3A%20%23999999%3B%0A%7D%0Aspan%2Eheader%2Dsection%2Dnumber%20%7B%0Apadding%2Dright%3A%201em%3B%0A%7D%0Aspan%2Etoc%2Dsection%2Dnumber%3A%3Aafter%20%7B%0Acontent%3A%20%22%20%22%3B%0Awhite%2Dspace%3A%20pre%3B%0A%7D%0A%40media%20print%20%7B%0A%2A%20%7B%0Abackground%3A%20transparent%20%21important%3B%0Acolor%3A%20black%20%21important%3B%0Afilter%3Anone%20%21important%3B%0A%2Dms%2Dfilter%3A%20none%20%21important%3B%0A%7D%0Abody%20%7B%0Afont%2Dsize%3A12pt%3B%0Amax%2Dwidth%3A100%25%3B%0A%7D%0Aa%2C%20a%3Avisited%20%7B%0Atext%2Ddecoration%3A%20underline%3B%0A%7D%0Ahr%20%7B%0Avisibility%3A%20hidden%3B%0Apage%2Dbreak%2Dbefore%3A%20always%3B%0A%7D%0Apre%2C%20blockquote%20%7B%0Apadding%2Dright%3A%201em%3B%0Apage%2Dbreak%2Dinside%3A%20avoid%3B%0A%7D%0Atr%2C%20img%20%7B%0Apage%2Dbreak%2Dinside%3A%20avoid%3B%0A%7D%0Aimg%20%7B%0Amax%2Dwidth%3A%20100%25%20%21important%3B%0A%7D%0A%40page%20%3Aleft%20%7B%0Amargin%3A%2015mm%2020mm%2015mm%2010mm%3B%0A%7D%0A%40page%20%3Aright%20%7B%0Amargin%3A%2015mm%2010mm%2015mm%2020mm%3B%0A%7D%0Ap%2C%20h2%2C%20h3%20%7B%0Aorphans%3A%203%3B%20widows%3A%203%3B%0A%7D%0Ah2%2C%20h3%20%7B%0Apage%2Dbreak%2Dafter%3A%20avoid%3B%0A%7D%0A%7D%0A" rel="stylesheet" type="text/css" />

<script type="text/javascript">
document.addEventListener("DOMContentLoaded", function() {
  var links = document.links;  
  for (var i = 0, linksLength = links.length; i < linksLength; i++)
    if(links[i].hostname != window.location.hostname)
      links[i].target = '_blank';
});
</script>

</head>

<body>


<div id="header">
<h1 class="title">Example report using bumphunter results</h1>
<h4 class="author"><em>L Collado-Torres</em></h4>
<h4 class="date"><em>11 April 2016</em></h4>
</div>
<h4 class="package">Package version: <span style="font-weight: normal">regionReport 1.5.45</span></h4>

<h1>Contents</h1>
<div id="TOC">
<ul>
<li><a href="#bumphunter-example"><span class="toc-section-number">1</span> <em><a href="http://bioconductor.org/packages/bumphunter">bumphunter</a></em> example</a></li>
<li><a href="#reproducibility"><span class="toc-section-number">2</span> Reproducibility</a></li>
</ul>
</div>

<div id="bumphunter-example" class="section level1">
<h1><span class="header-section-number">1</span> <em><a href="http://bioconductor.org/packages/bumphunter">bumphunter</a></em> example</h1>
<p>The <em><a href="http://bioconductor.org/packages/bumphunter">bumphunter</a></em> package can be used for methylation analyses where you are interested in identifying differentially methylated regions. The <a href="http://bioconductor.org/packages/release/bioc/vignettes/bumphunter/inst/doc/bumphunter.pdf">vignette</a> explains in greater detail the data set we are using in this example.</p>
<pre class="r"><code>## Load bumphunter
library('bumphunter')</code></pre>
<pre><code>## Loading required package: S4Vectors</code></pre>
<pre><code>## Loading required package: stats4</code></pre>
<pre><code>## Loading required package: BiocGenerics</code></pre>
<pre><code>## Loading required package: parallel</code></pre>
<pre><code>## 
## Attaching package: 'BiocGenerics'</code></pre>
<pre><code>## The following objects are masked from 'package:parallel':
## 
##     clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,
##     clusterExport, clusterMap, parApply, parCapply, parLapply,
##     parLapplyLB, parRapply, parSapply, parSapplyLB</code></pre>
<pre><code>## The following objects are masked from 'package:stats':
## 
##     IQR, mad, xtabs</code></pre>
<pre><code>## The following objects are masked from 'package:base':
## 
##     anyDuplicated, append, as.data.frame, cbind, colnames,
##     do.call, duplicated, eval, evalq, Filter, Find, get, grep,
##     grepl, intersect, is.unsorted, lapply, lengths, Map, mapply,
##     match, mget, order, paste, pmax, pmax.int, pmin, pmin.int,
##     Position, rank, rbind, Reduce, rownames, sapply, setdiff,
##     sort, table, tapply, union, unique, unsplit</code></pre>
<pre><code>## 
## Attaching package: 'S4Vectors'</code></pre>
<pre><code>## The following objects are masked from 'package:base':
## 
##     colMeans, colSums, expand.grid, rowMeans, rowSums</code></pre>
<pre><code>## Loading required package: IRanges</code></pre>
<pre><code>## Loading required package: GenomeInfoDb</code></pre>
<pre><code>## Loading required package: GenomicRanges</code></pre>
<pre><code>## Loading required package: foreach</code></pre>
<pre><code>## foreach: simple, scalable parallel programming from Revolution Analytics
## Use Revolution R for scalability, fault tolerance and more.
## http://www.revolutionanalytics.com</code></pre>
<pre><code>## Loading required package: iterators</code></pre>
<pre><code>## Loading required package: locfit</code></pre>
<pre><code>## locfit 1.5-9.1    2013-03-22</code></pre>
<pre class="r"><code>## Create data from the vignette
pos &lt;- list(pos1=seq(1, 1000, 35),
            pos2=seq(2001, 3000, 35),
            pos3=seq(1, 1000, 50))
chr &lt;- rep(paste0('chr', c(1, 1, 2)), times = sapply(pos, length))
pos &lt;- unlist(pos, use.names = FALSE)

## Find clusters
cl &lt;- clusterMaker(chr, pos, maxGap = 300)

## Build simulated bumps
Indexes &lt;- split(seq_along(cl), cl)
beta1 &lt;- rep(0, length(pos))
for(i in seq(along=Indexes)){
    ind &lt;- Indexes[[i]]
    x &lt;- pos[ind]
    z &lt;- scale(x, median(x), max(x)/12)
    beta1[ind] &lt;- i*(-1)^(i+1)*pmax(1-abs(z)^3,0)^3 ##multiply by i to vary size
}

## Build data
beta0 &lt;- 3 * sin(2 * pi * pos / 720)
X &lt;- cbind(rep(1, 20), rep(c(0, 1), each = 10))
set.seed(23852577)
error &lt;- matrix(rnorm(20 * length(beta1), 0, 1), ncol = 20)
y &lt;- t(X[, 1]) %x% beta0 + t(X[, 2]) %x% beta1 + error

## Perform bumphunting
tab &lt;- bumphunter(y, X, chr, pos, cl, cutoff=.5)</code></pre>
<pre><code>## [bumphunterEngine] Using a single core (backend: doSEQ, version: 1.4.3).</code></pre>
<pre><code>## [bumphunterEngine] Computing coefficients.</code></pre>
<pre><code>## [bumphunterEngine] Finding regions.</code></pre>
<pre><code>## [bumphunterEngine] Found 15 bumps.</code></pre>
<pre class="r"><code>## Explore data
lapply(tab, head)</code></pre>
<pre><code>## $table
##     chr start  end      value       area cluster indexStart indexEnd  L
## 10 chr1  2316 2631 -1.5814747 15.8147473       2         39       48 10
## 7  chr2   451  551  1.5891293  4.7673878       3         68       70  3
## 2  chr1   456  526  1.0678828  3.2036485       1         14       16  3
## 5  chr1  2176 2211  0.7841794  1.5683589       2         35       36  2
## 6  chr1  2841 2841  1.2010184  1.2010184       2         54       54  1
## 4  chr1   771  771  0.7780902  0.7780902       1         23       23  1
##    clusterL
## 10       29
## 7        20
## 2        29
## 5        29
## 6        29
## 4        29
## 
## $coef
##             [,1]
## [1,]  0.60960932
## [2,] -0.09052769
## [3,] -0.21482638
## [4,]  0.13053755
## [5,] -0.21723642
## [6,]  0.39934961
## 
## $fitted
##             [,1]
## [1,]  0.60960932
## [2,] -0.09052769
## [3,] -0.21482638
## [4,]  0.13053755
## [5,] -0.21723642
## [6,]  0.39934961
## 
## $pvaluesMarginal
## [1] NA</code></pre>
<p>Once we have the regions we can proceed to build the required <code>GRanges</code> object.</p>
<pre class="r"><code>library('GenomicRanges')

## Build GRanges with sequence lengths
regions &lt;- GRanges(seqnames = tab$table$chr, 
    IRanges(start = tab$table$start, end = tab$table$end),
    strand = '*', value = tab$table$value, area = tab$table$area, 
    cluster = tab$table$cluster, L = tab$table$L, clusterL = tab$table$clusterL)

## Assign chr lengths
data(hg19Ideogram, package = 'biovizBase')
seqlengths(regions) &lt;- seqlengths(hg19Ideogram)[names(seqlengths(regions))]

## Explore the regions
regions</code></pre>
<pre><code>## GRanges object with 15 ranges and 5 metadata columns:
##        seqnames       ranges strand |              value              area
##           &lt;Rle&gt;    &lt;IRanges&gt;  &lt;Rle&gt; |          &lt;numeric&gt;         &lt;numeric&gt;
##    [1]     chr1 [2316, 2631]      * |  -1.58147472668612  15.8147472668612
##    [2]     chr2 [ 451,  551]      * |   1.58912926544627  4.76738779633882
##    [3]     chr1 [ 456,  526]      * |   1.06788283103948  3.20364849311844
##    [4]     chr1 [2176, 2211]      * |  0.784179442360784  1.56835888472157
##    [5]     chr1 [2841, 2841]      * |   1.20101842543356  1.20101842543356
##    ...      ...          ...    ... .                ...               ...
##   [11]     chr1 [ 631,  631]      * |  0.618602646982714 0.618602646982714
##   [12]     chr1 [   1,    1]      * |  0.609609318450113 0.609609318450113
##   [13]     chr1 [2911, 2911]      * | -0.576423040796044 0.576423040796044
##   [14]     chr2 [ 251,  251]      * | -0.556159535339166 0.556159535339166
##   [15]     chr1 [2806, 2806]      * | -0.521605772372843 0.521605772372843
##          cluster         L  clusterL
##        &lt;numeric&gt; &lt;numeric&gt; &lt;integer&gt;
##    [1]         2        10        29
##    [2]         3         3        20
##    [3]         1         3        29
##    [4]         2         2        29
##    [5]         2         1        29
##    ...       ...       ...       ...
##   [11]         1         1        29
##   [12]         1         1        29
##   [13]         2         1        29
##   [14]         3         1        20
##   [15]         2         1        29
##   -------
##   seqinfo: 2 sequences from an unspecified genome</code></pre>
<p>Now that we have identified a set of differentially methylated regions we can proceed to creating the HTML report. Note that this report has less information than the <a href="http://leekgroup.github.io/regionReportSupp/DiffBind.html">DiffBind example</a> because we donâ€™t have a p-value variable.</p>
<pre class="r"><code>## Load regionReport
library('regionReport')</code></pre>
<pre class="r"><code>## Make it so that the report will be available as a vignette
original &lt;- readLines(system.file('regionExploration', 'regionExploration.Rmd',
    package = 'regionReport'))
vignetteInfo &lt;- c(
    'vignette: &gt;',
    '  %\\VignetteEngine{knitr::rmarkdown}',
    '  %\\VignetteIndexEntry{Basic genomic regions exploration}',
    '  %\\VignetteEncoding{UTF-8}'
)
new &lt;- c(original[1:12], vignetteInfo, original[13:length(original)])
writeLines(new, '/Users/lcollado/Dropbox/JHSPH/Code/regionReportSupp/bumphunter-example/regionReportBumphunter.Rmd')

## Now create the report
report &lt;- renderReport(regions, 'Example bumphunter', pvalueVars = NULL,
    densityVars = c('Area' = 'area', 'Value' = 'value',
    'Cluster Length' = 'clusterL'), significantVar = NULL,
    output = 'index', outdir = 'bumphunter-example',
    template = '/Users/lcollado/Dropbox/JHSPH/Code/regionReportSupp/bumphunter-example/regionReportBumphunter.Rmd', device = 'png')</code></pre>
<pre><code>## Warning: replacing previous import 'ggplot2::Position' by
## 'BiocGenerics::Position' when loading 'ggbio'</code></pre>
<pre><code>## Writing 11 Bibtex entries ...</code></pre>
<pre><code>## OK
## Results written to file 'bumphunter-example/index.bib'</code></pre>
<pre><code>## 
## 
## processing file: index.Rmd</code></pre>
<pre><code>## 
  |                                                                       
  |                                                                 |   0%
  |                                                                       
  |..                                                               |   2%
##    inline R code fragments
## 
## 
  |                                                                       
  |...                                                              |   5%
## label: docSetup (with options) 
## List of 3
##  $ bootstrap.show.code   : logi FALSE
##  $ dev                   : symbol device
##  $ bootstrap.show.message: logi FALSE
## 
## 
  |                                                                       
  |.....                                                            |   7%
##   ordinary text without R code
## 
## 
  |                                                                       
  |......                                                           |   9%
## label: setup (with options) 
## List of 1
##  $ bootstrap.show.message: logi FALSE
## 
## 
  |                                                                       
  |........                                                         |  12%
##   ordinary text without R code
## 
## 
  |                                                                       
  |.........                                                        |  14%
## label: pvaluePlots (with options) 
## List of 3
##  $ echo   : logi FALSE
##  $ results: chr &quot;asis&quot;
##  $ eval   : symbol hasPvalueVars
## 
## 
  |                                                                       
  |...........                                                      |  16%
##   ordinary text without R code
## 
## 
  |                                                                       
  |............                                                     |  19%
## label: regLen (with options) 
## List of 4
##  $ fig.width : num 14
##  $ fig.height: num 14
##  $ eval      : symbol hasSignificant
##  $ echo      : symbol hasSignificant
## 
## 
  |                                                                       
  |..............                                                   |  21%
##   ordinary text without R code
## 
## 
  |                                                                       
  |...............                                                  |  23%
## label: regLen2 (with options) 
## List of 4
##  $ fig.width : num 10
##  $ fig.height: num 10
##  $ eval      : language !hasSignificant
##  $ echo      : language !hasSignificant</code></pre>
<pre><code>## 
  |                                                                       
  |.................                                                |  26%
##    inline R code fragments
## 
## 
  |                                                                       
  |..................                                               |  28%
## label: densityPlots (with options) 
## List of 3
##  $ echo   : logi FALSE
##  $ results: chr &quot;asis&quot;
##  $ eval   : symbol hasDensityVars</code></pre>
<pre><code>## 
  |                                                                       
  |....................                                             |  30%
##    inline R code fragments
## 
## 
  |                                                                       
  |.....................                                            |  33%
## label: genomeOverview1 (with options) 
## List of 6
##  $ message   : logi FALSE
##  $ fig.width : num 7
##  $ fig.height: num 9
##  $ dpi       : num 300
##  $ eval      : symbol hasSignificant
##  $ echo      : symbol hasSignificant
## 
## 
  |                                                                       
  |.......................                                          |  35%
##    inline R code fragments
## 
## 
  |                                                                       
  |........................                                         |  37%
## label: manhattanPlots (with options) 
## List of 3
##  $ echo   : logi FALSE
##  $ results: chr &quot;asis&quot;
##  $ eval   : symbol hasPvalueVars
## 
## 
  |                                                                       
  |..........................                                       |  40%
##   ordinary text without R code
## 
## 
  |                                                                       
  |...........................                                      |  42%
## label: genomeOverview2 (with options) 
## List of 4
##  $ message   : logi FALSE
##  $ fig.width : num 7
##  $ fig.height: num 9
##  $ dpi       : num 300</code></pre>
<pre><code>## No annotationPackage supplied. Trying org.Hs.eg.db.</code></pre>
<pre><code>## Loading required package: org.Hs.eg.db</code></pre>
<pre><code>## Loading required package: DBI</code></pre>
<pre><code>## </code></pre>
<pre><code>## Getting TSS and TSE.</code></pre>
<pre><code>## Getting CSS and CSE.</code></pre>
<pre><code>## Getting exons.</code></pre>
<pre><code>## Annotating genes.</code></pre>
<pre><code>## Scale for 'x' is already present. Adding another scale for 'x', which
## will replace the existing scale.</code></pre>
<pre><code>## 
  |                                                                       
  |.............................                                    |  44%
##    inline R code fragments
## 
## 
  |                                                                       
  |..............................                                   |  47%
## label: annoReg (with options) 
## List of 1
##  $ results: chr &quot;asis&quot;
## 
## 
  |                                                                       
  |................................                                 |  49%
##   ordinary text without R code
## 
## 
  |                                                                       
  |.................................                                |  51%
## label: genomeOverview3 (with options) 
## List of 6
##  $ message   : logi FALSE
##  $ fig.width : num 7
##  $ fig.height: num 9
##  $ dpi       : num 300
##  $ eval      : symbol hasSignificant
##  $ echo      : symbol hasSignificant
## 
## 
  |                                                                       
  |...................................                              |  53%
##    inline R code fragments
## 
## 
  |                                                                       
  |....................................                             |  56%
## label: countTable (with options) 
## List of 1
##  $ results: chr &quot;asis&quot;
## 
## 
  |                                                                       
  |......................................                           |  58%
##   ordinary text without R code
## 
## 
  |                                                                       
  |.......................................                          |  60%
## label: vennDiagram (with options) 
## List of 2
##  $ fig.width : num 7
##  $ fig.height: num 7</code></pre>
<pre><code>## 
  |                                                                       
  |.........................................                        |  63%
##    inline R code fragments
## 
## 
  |                                                                       
  |..........................................                       |  65%
## label: vennDiagramSignificant (with options) 
## List of 4
##  $ eval      : symbol hasSignificant
##  $ echo      : symbol hasSignificant
##  $ fig.width : num 7
##  $ fig.height: num 7
## 
## 
  |                                                                       
  |............................................                     |  67%
##    inline R code fragments
## 
## 
  |                                                                       
  |.............................................                    |  70%
## label: bestRegionInfo (with options) 
## List of 1
##  $ results: chr &quot;asis&quot;
## 
## 
  |                                                                       
  |...............................................                  |  72%
##   ordinary text without R code
## 
## 
  |                                                                       
  |................................................                 |  74%
## label: unnamed-chunk-1 (with options) 
## List of 2
##  $ child: symbol customCode
##  $ eval : symbol hasCustomCode
## 
## 
  |                                                                       
  |..................................................               |  77%
##    inline R code fragments
## 
## 
  |                                                                       
  |...................................................              |  79%
## label: thecall (with options) 
## List of 1
##  $ echo: logi FALSE
## 
## 
  |                                                                       
  |.....................................................            |  81%
##   ordinary text without R code
## 
## 
  |                                                                       
  |......................................................           |  84%
## label: reproducibility1 (with options) 
## List of 1
##  $ echo: logi FALSE
## 
## 
  |                                                                       
  |........................................................         |  86%
##   ordinary text without R code
## 
## 
  |                                                                       
  |.........................................................        |  88%
## label: reproducibility2 (with options) 
## List of 1
##  $ echo: logi FALSE
## 
## 
  |                                                                       
  |...........................................................      |  91%
##   ordinary text without R code
## 
## 
  |                                                                       
  |............................................................     |  93%
## label: reproducibility3 (with options) 
## List of 1
##  $ echo: logi FALSE
## 
## 
  |                                                                       
  |..............................................................   |  95%
##    inline R code fragments
## 
## 
  |                                                                       
  |...............................................................  |  98%
## label: bibliography (with options) 
## List of 3
##  $ results: chr &quot;asis&quot;
##  $ echo   : logi FALSE
##  $ warning: logi FALSE
## 
## 
  |                                                                       
  |.................................................................| 100%
##   ordinary text without R code</code></pre>
<pre><code>## output file: index.knit.md</code></pre>
<pre><code>## /usr/local/bin/pandoc +RTS -K512m -RTS index.utf8.md --to html --from markdown+autolink_bare_uris+ascii_identifiers+tex_math_single_backslash-implicit_figures --output index.html --smart --email-obfuscation none --self-contained --standalone --section-divs --table-of-contents --toc-depth 3 --variable toc_float=1 --variable toc_selectors=h1,h2,h3 --variable toc_collapsed=1 --variable toc_smooth_scroll=1 --template /Library/Frameworks/R.framework/Versions/3.3/Resources/library/rmarkdown/rmd/h/default.html --variable 'theme:spacelab' --include-in-header /var/folders/cx/n9s558kx6fb7jf5z_pgszgb80000gn/T//RtmpgGUgnB/rmarkdown-str2e5a5e4f99fc.html --mathjax --variable 'mathjax-url:https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML' --no-highlight --variable highlightjs=/Library/Frameworks/R.framework/Versions/3.3/Resources/library/rmarkdown/rmd/h/highlight --variable navigationjs=/Library/Frameworks/R.framework/Versions/3.3/Resources/library/rmarkdown/rmd/h/navigation-1.0 --variable code_folding=hide</code></pre>
<pre><code>## 
## Output created: index.html</code></pre>
<pre class="r"><code>## Clean up
file.remove('/Users/lcollado/Dropbox/JHSPH/Code/regionReportSupp/bumphunter-example/regionReportBumphunter.Rmd')</code></pre>
<pre><code>## [1] TRUE</code></pre>
<p>You can view the final report <a href="/regionReportSupp/bumphunter-example/index.html">here</a>.</p>
</div>
<div id="reproducibility" class="section level1">
<h1><span class="header-section-number">2</span> Reproducibility</h1>
<pre class="r"><code>## Date generated:
Sys.time()</code></pre>
<pre><code>## [1] &quot;2016-04-11 22:14:41 EDT&quot;</code></pre>
<pre class="r"><code>## Time spent making this page:
proc.time()</code></pre>
<pre><code>##    user  system elapsed 
## 163.543   7.108 173.534</code></pre>
<pre class="r"><code>## R and packages info:
options(width = 120)
devtools::session_info()</code></pre>
<pre><code>## Session info -----------------------------------------------------------------------------------------------------------</code></pre>
<pre><code>##  setting  value                                    
##  version  R version 3.3.0 alpha (2016-03-23 r70368)
##  system   x86_64, darwin13.4.0                     
##  ui       AQUA                                     
##  language (EN)                                     
##  collate  en_US.UTF-8                              
##  tz       America/New_York                         
##  date     2016-04-11</code></pre>
<pre><code>## Packages ---------------------------------------------------------------------------------------------------------------</code></pre>
<pre><code>##  package                           * version  date       source                                   
##  acepack                             1.3-3.3  2014-11-24 CRAN (R 3.3.0)                           
##  annotate                            1.49.1   2016-02-06 Bioconductor                             
##  AnnotationDbi                     * 1.33.8   2016-04-10 Bioconductor                             
##  AnnotationHub                       2.3.16   2016-03-25 Bioconductor                             
##  backports                           1.0.2    2016-03-18 CRAN (R 3.3.0)                           
##  bibtex                              0.4.0    2014-12-31 CRAN (R 3.3.0)                           
##  Biobase                           * 2.31.3   2016-01-14 Bioconductor                             
##  BiocGenerics                      * 0.17.4   2016-04-07 Bioconductor                             
##  BiocInstaller                       1.21.4   2016-03-23 Bioconductor                             
##  BiocParallel                        1.5.21   2016-03-23 Bioconductor                             
##  BiocStyle                         * 1.99.0   2016-04-05 Bioconductor                             
##  biomaRt                             2.27.2   2016-01-14 Bioconductor                             
##  Biostrings                          2.39.12  2016-02-21 Bioconductor                             
##  biovizBase                          1.19.6   2016-04-06 Bioconductor                             
##  bitops                              1.0-6    2013-08-17 CRAN (R 3.3.0)                           
##  BSgenome                            1.39.4   2016-02-21 Bioconductor                             
##  bumphunter                        * 1.11.5   2016-03-29 Bioconductor                             
##  checkmate                           1.7.4    2016-04-08 CRAN (R 3.3.0)                           
##  cluster                             2.0.3    2015-07-21 CRAN (R 3.3.0)                           
##  codetools                           0.2-14   2015-07-15 CRAN (R 3.3.0)                           
##  colorspace                          1.2-6    2015-03-11 CRAN (R 3.3.0)                           
##  DBI                               * 0.3.1    2014-09-24 CRAN (R 3.3.0)                           
##  DEFormats                           0.99.8   2016-03-31 Bioconductor                             
##  derfinder                         * 1.5.30   2016-03-25 Bioconductor                             
##  derfinderHelper                     1.5.3    2016-03-23 Bioconductor                             
##  derfinderPlot                     * 1.5.7    2016-03-23 Bioconductor                             
##  DESeq2                              1.11.42  2016-04-10 Bioconductor                             
##  devtools                          * 1.10.0   2016-01-23 CRAN (R 3.3.0)                           
##  dichromat                           2.0-0    2013-01-24 CRAN (R 3.3.0)                           
##  digest                              0.6.9    2016-01-08 CRAN (R 3.3.0)                           
##  doRNG                               1.6      2014-03-07 CRAN (R 3.3.0)                           
##  DT                                * 0.1      2015-06-09 CRAN (R 3.3.0)                           
##  edgeR                               3.13.8   2016-04-08 Bioconductor                             
##  ensembldb                           1.3.19   2016-04-03 Bioconductor                             
##  evaluate                            0.8.3    2016-03-05 CRAN (R 3.3.0)                           
##  foreach                           * 1.4.3    2015-10-13 CRAN (R 3.3.0)                           
##  foreign                             0.8-66   2015-08-19 CRAN (R 3.3.0)                           
##  formatR                             1.3      2016-03-05 CRAN (R 3.3.0)                           
##  Formula                             1.2-1    2015-04-07 CRAN (R 3.3.0)                           
##  genefilter                          1.53.3   2016-03-23 Bioconductor                             
##  geneplotter                         1.49.0   2016-01-14 Bioconductor                             
##  GenomeInfoDb                      * 1.7.6    2016-01-29 Bioconductor                             
##  GenomicAlignments                   1.7.20   2016-02-25 Bioconductor                             
##  GenomicFeatures                   * 1.23.29  2016-04-05 Bioconductor                             
##  GenomicFiles                        1.7.9    2016-02-22 Bioconductor                             
##  GenomicRanges                     * 1.23.25  2016-03-31 Bioconductor                             
##  GGally                              1.0.1    2016-01-14 CRAN (R 3.3.0)                           
##  ggbio                             * 1.19.13  2016-04-03 Bioconductor                             
##  ggplot2                           * 2.1.0    2016-03-01 CRAN (R 3.3.0)                           
##  graph                               1.49.1   2016-01-14 Bioconductor                             
##  gridExtra                         * 2.2.1    2016-02-29 CRAN (R 3.3.0)                           
##  gtable                              0.2.0    2016-02-26 CRAN (R 3.3.0)                           
##  highr                               0.5.1    2015-09-18 CRAN (R 3.3.0)                           
##  Hmisc                               3.17-3   2016-04-03 CRAN (R 3.3.0)                           
##  htmltools                           0.3.5    2016-03-21 CRAN (R 3.3.0)                           
##  htmlwidgets                         0.6      2016-02-25 CRAN (R 3.3.0)                           
##  httpuv                              1.3.3    2015-08-04 CRAN (R 3.3.0)                           
##  httr                                1.1.0    2016-01-28 CRAN (R 3.3.0)                           
##  interactiveDisplayBase              1.9.0    2016-01-14 Bioconductor                             
##  IRanges                           * 2.5.43   2016-04-10 Bioconductor                             
##  iterators                         * 1.0.8    2015-10-13 CRAN (R 3.3.0)                           
##  jsonlite                            0.9.19   2015-11-28 CRAN (R 3.3.0)                           
##  knitcitations                       1.0.7    2015-10-28 CRAN (R 3.3.0)                           
##  knitr                             * 1.12.3   2016-01-22 CRAN (R 3.3.0)                           
##  knitrBootstrap                      1.0.0    2016-03-24 Github (jimhester/knitrBootstrap@cdaa4a9)
##  labeling                            0.3      2014-08-23 CRAN (R 3.3.0)                           
##  lattice                             0.20-33  2015-07-14 CRAN (R 3.3.0)                           
##  latticeExtra                        0.6-28   2016-02-09 CRAN (R 3.3.0)                           
##  limma                               3.27.14  2016-03-23 Bioconductor                             
##  locfit                            * 1.5-9.1  2013-04-20 CRAN (R 3.3.0)                           
##  lubridate                           1.5.6    2016-04-06 CRAN (R 3.3.0)                           
##  magrittr                            1.5      2014-11-22 CRAN (R 3.3.0)                           
##  markdown                            0.7.7    2015-04-22 CRAN (R 3.3.0)                           
##  Matrix                              1.2-4    2016-03-02 CRAN (R 3.3.0)                           
##  matrixStats                         0.50.1   2015-12-15 CRAN (R 3.3.0)                           
##  memoise                             1.0.0    2016-01-29 CRAN (R 3.3.0)                           
##  mgcv                              * 1.8-12   2016-03-03 CRAN (R 3.3.0)                           
##  mime                                0.4      2015-09-03 CRAN (R 3.3.0)                           
##  munsell                             0.4.3    2016-02-13 CRAN (R 3.3.0)                           
##  nlme                              * 3.1-126  2016-03-14 CRAN (R 3.3.0)                           
##  nnet                                7.3-12   2016-02-02 CRAN (R 3.3.0)                           
##  org.Hs.eg.db                      * 3.3.0    2016-04-11 Bioconductor                             
##  OrganismDbi                         1.13.6   2016-04-05 Bioconductor                             
##  pkgmaker                            0.22     2014-05-14 CRAN (R 3.3.0)                           
##  plyr                                1.8.3    2015-06-12 CRAN (R 3.3.0)                           
##  qvalue                              2.3.2    2016-01-14 Bioconductor                             
##  R6                                  2.1.2    2016-01-26 CRAN (R 3.3.0)                           
##  RBGL                                1.47.0   2016-01-14 Bioconductor                             
##  RColorBrewer                      * 1.1-2    2014-12-07 CRAN (R 3.3.0)                           
##  Rcpp                                0.12.4   2016-03-26 CRAN (R 3.3.0)                           
##  RCurl                               1.95-4.8 2016-03-01 CRAN (R 3.3.0)                           
##  RefManageR                          0.10.13  2016-04-04 CRAN (R 3.3.0)                           
##  regionReport                      * 1.5.45   2016-04-12 Bioconductor                             
##  registry                            0.3      2015-07-08 CRAN (R 3.3.0)                           
##  reshape                             0.8.5    2014-04-23 CRAN (R 3.3.0)                           
##  reshape2                            1.4.1    2014-12-06 CRAN (R 3.3.0)                           
##  RJSONIO                             1.3-0    2014-07-28 CRAN (R 3.3.0)                           
##  rmarkdown                           0.9.5    2016-02-22 CRAN (R 3.3.0)                           
##  rngtools                            1.2.4    2014-03-06 CRAN (R 3.3.0)                           
##  rpart                               4.1-10   2015-06-29 CRAN (R 3.3.0)                           
##  Rsamtools                           1.23.8   2016-04-10 Bioconductor                             
##  RSQLite                           * 1.0.0    2014-10-25 CRAN (R 3.3.0)                           
##  rstudioapi                          0.5      2016-01-24 CRAN (R 3.3.0)                           
##  rtracklayer                         1.31.10  2016-04-07 Bioconductor                             
##  S4Vectors                         * 0.9.46   2016-04-07 Bioconductor                             
##  scales                              0.4.0    2016-02-26 CRAN (R 3.3.0)                           
##  shiny                               0.13.2   2016-03-28 CRAN (R 3.3.0)                           
##  stringi                             1.0-1    2015-10-22 CRAN (R 3.3.0)                           
##  stringr                             1.0.0    2015-04-30 CRAN (R 3.3.0)                           
##  SummarizedExperiment                1.1.23   2016-04-06 Bioconductor                             
##  survival                            2.38-3   2015-07-02 CRAN (R 3.3.0)                           
##  TxDb.Hsapiens.UCSC.hg19.knownGene * 3.2.2    2016-03-24 Bioconductor                             
##  VariantAnnotation                   1.17.23  2016-04-07 Bioconductor                             
##  whisker                           * 0.3-2    2013-04-28 CRAN (R 3.3.0)                           
##  XML                                 3.98-1.4 2016-03-01 CRAN (R 3.3.0)                           
##  xtable                              1.8-2    2016-02-05 CRAN (R 3.3.0)                           
##  XVector                             0.11.8   2016-04-06 Bioconductor                             
##  yaml                                2.1.13   2014-06-12 CRAN (R 3.3.0)                           
##  zlibbioc                            1.17.1   2016-03-19 Bioconductor</code></pre>
</div>



<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
